### ISSUE TITLE
Get-ITGlueConfigurations -organization_id not working

### ISSUE THREAD

**Issue by adrianwells** [itglue/powershellwrapper#21]
@CalebAlbers - working with https://github.com/CalebAlbers/powershellwrapper the Get-ITGlueConfigurations -organization_id not does not appear to be working when specifying an organization ID.  Trying to figure out why and thought I'd put an issue out here for it (I cannot put issues against https://github.com/CalebAlbers/powershellwrapper).

The issue is that all organizations are returned and parameter appears to be ignored.

**Comment by ecspresso** [comment]
Default ParameterSetName is "index", `$organizationd-id` matches both "index" and "show" and with no other parameter present, `$PSCmdlet.ParameterSetName` will be "index" and the URI will not be the nested version.

When setting $organization-id, one will always want the nested relationships route, perhaps change to this
```posh
[Parameter(ParameterSetName = 'index')]
[Parameter(ParameterSetName = 'show')]
[Nullable[Int64]]$organization_id = $null,
```
to this
```posh
[Parameter(ParameterSetName = 'show')]
[Nullable[Int64]]$organization_id = $null,
```

**Comment by CalebAlbers** [comment]
Good catch!

According to the [API documentation](https://api.itglue.com/developer/), both Index and Show for the Configurations endpoint allow for nested relationships route.

Currently, the nested relationships route in the `Get-ITGlueConfigurations` function is only set to be enabled with `$PSCmdlet.ParameterSetName` is `show` (as @ecspresso mentioned). I'm going to change this so that the nested relationships route is enabled whenever `$organization_id` is not `$null`.

Commit coming in a couple of minutes. I'm also going to create a development branch on this repo and push all of the changes that I have on my own repo over to it. That should make collaboration more manageable, as pull requests can be made directly to the development branch without worrying about Master.

**Comment by adrianwells** [comment]
@CalebAlbers and @ecspresso - if I understand correctly then this should be resolved with the change starting [here](https://github.com/itglue/powershellwrapper/compare/development#diff-1c73df89cf1645492d18ff338ad112a1R85), correct?  If yes, then my testing on the development branch this is still not providing the expected results when passing a valid parameter via
`Get-ITGlueConfigurations -organization_id`

Similar experience with
`Get-ITGlueConfigurations -filter_organization_id`

It appears the function returns more than one organization and is still not working as expected with the parameters.

Trying to determine why and assistance is welcome.

**Comment by CalebAlbers** [comment]
@adrianwells @ecspresso This is now working properly (for me) with the updated code in the #28 PR. If you don't mind, can you test the new code? I will keep this open and will wait on merging that PR until this is for sure resolved.

![screen shot 2018-09-09 at 11 11 27 pm](https://user-images.githubusercontent.com/7110138/45276048-c2674d00-b485-11e8-8e23-d04329edff02.png)

**Comment by ecspresso** [comment]
Can't find any problem when testing on my end.

**Comment by CalebAlbers** [comment]
#28 has been merged with the v2.0.0 release. Closing this.

Thanks!
